test/categories.test.js

```diff
@@ -0,0 +1,166 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../api/app';
+import { Category, allCategories } from '../api/v1/models/Category';
+
+const should = chai.should();
+chai.use(chaiHttp);
+
+const initialCategory = new Category({
+  id: 1,
+  name: 'Salads',
+  description: 'Healthy!!!',
+  updatedAt: '03-09-2018',
+  createdAt: '03-09-2018',
+});
+
+// Initialize test database for test
+beforeEach((done) => {
+  chai.request(app)
+    .post('/api/v1/categories')
+    .send(initialCategory)
+    .end((err, res) => {
+      if (err) return done(err);
+      done();
+    });
+});
+
+afterEach((done) => {
+  allCategories.length = 0;
+  done();
+});
+
+describe('Food Categories', () => {
+  describe('GET /categories - Get all food item categories', () => {
+    it('should return a json object', (done) => {
+      chai.request(app)
+        .get('/api/v1/categories')
+        .end((err, res) => {
+          res.should.have.status(200);
+          expect(res).to.be.a.json;
+          done();
```

Comment:
Assert that the array of items you are expecting is actually returned. 
Hint: import the array of items into this file. Assert that the array in the response body matches the imported array